### ISSUE TITLE
FFMLP(fp-16) not compatable with Jittor=13.8.5

### ISSUE THREAD

**Issue by RobotDogCyberslacking** [Jittor/JNeRF#73]
following the instructions in ReadMe.md , I got this when running the ngp_fox test:
```
/hy-tmp/JNeRF-master/python/jnerf/runner/runner.py:193: RuntimeWarning: invalid value encountered in cast
  ndarr = (img*255+0.5).clip(0, 255).astype('uint8')
```

The output image in log folder is all black.
Some loss value is NAN.

my hardware environment:
Intel(R) Xeon(R) CPU E5-2686 v4
RTX3090-24G

The training and output is all correct when I set using FP_16 = False in config file, which make the code use the MLP from pytorch.nn instead of FMLP.

I tried to fix it by downgrading my python to 3.8, jittor=1.3.4.13(exactly the version in requirements.txt),without running python setup.py, and everything goes fine.

If you encounter the same problem, try:

1. downgrade your python to 3.8, jittor=1.3.4.13
2. do not run python setup.py again, because it will upgrade your jittor package to the latest version automaticly.

I hope you guys fix it ASAP.

**Comment by scymz2** [comment]
jittor=1.3.4.13 doesn't support -arch=compute_89 (40-series GPU), need to mannually update cuda >=11.8,  but I don't know how to do that.
